.github/workflows/ci.yml

```diff
@@ -17,6 +17,10 @@ env:
 permissions:
   contents: read
 
+concurrency:
+  group: ${{ github.workflow }}-${{ github.head_ref || github.run_id }}
+  cancel-in-progress: true
```

Comment:
I will let the team decide on this one. Sometimes getting all runs is useful, because you can see when a failure was introduced.

Reply:
```suggestion
```